cmd/syncthing/main.go

```diff
@@ -912,27 +888,31 @@ func printHashRate() {
 	l.Infof("Single thread hash performance is ~%.*f MB/s", decimals, hashRate)
 }
 
-func loadConfig(cfgFile string) (*config.Wrapper, string, error) {
-	info, err := os.Stat(cfgFile)
-	if err != nil {
-		return nil, "", err
-	}
-	if !info.Mode().IsRegular() {
-		return nil, "", errors.New("configuration is not a file")
+func myDeviceName(cfg *config.Wrapper) string {
```

Comment:
Could be a pointer receiver func on the config. And we could refactor NewModel to call that internally, rather than passing it in.